package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "react-form",
-  "version": "4.0.1",
+  "version": "4.0.2",
   "description": "⚛️ 💼 React hooks for managing form state and lifecycle",
   "author": "tannerlinsley",
   "license": "MIT",
   "repository": "tannerlinsley/react-form",
   "main": "dist/index.js",
   "module": "dist/index.es.js",
-  "scripts": {
+  "toMakeItWorkRenameThisKeyToSingleWordScripts": {
```

Comment:
Without this change, when running `yarn` in Popmenu it will automatically call `prepare` script. Instead we add dist folder to git.